### PR TITLE
[TASK] Small corrections for the README

### PR DESCRIPTION
**Pull request by ayacoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,26 +13,27 @@ Once installed ,there appears a Deepl back end module with a settings tab.
 ## Requirements
 - TYPO3 8.5 to 8.7.99 (V1.0.0 - 1.0.1)
 - TYPO3 9.5.1 to 9.5.99 (V1.0.2 onwards)
+- TYPO3 9.5.1 to 10.4.99 (V1.1.2 onwards)
```
Comment (sypets) [inline README.md:16]: Do we really want to keep updating the requirements or just refer to extensions.typo3.org / Packagist?

File: README.md [hunk 2]
```diff
@@ -70,10 +71,10 @@ For the perfect working of deepltranslate with editor users , we need to make su
 
 
 ## Deepl Module Settings
-The settings module helps to assign the sytem languages to either deepl supported languages or to Google supported languages.
+The settings module helps to assign the system languages to either Deepl supported languages or to Google supported languages.
 
-For example you can assign German under Austrian German sys language if you wish. For assigning a language to a sys language you must enter it’s isocode(ISO 639-1).
+For example, you can assign German under Austrian German sys language if you wish. For assigning a language to a sys language you must enter it’s isocode (ISO 639-1).
```
Comment (sypets) [inline README.md:76]: Should be "its" here, not "it's".

**Review by sypets — COMMENTED** [review]
... the rest looks good :thumbsup: ... 

(One could still nitpick the language some more, but that would blow up this PR, should probably be merged first).

File: README.md (hunk 1 above)
Comment (ayacoo) [reply README.md:16]: I am used to the system requirements also simply being in the README. That's why I thought an update was ok here. What do you say?

File: README.md (hunk 2 above)
Comment (ayacoo) [reply README.md]: Thx and done :-)

File: README.md (hunk 1 above)
Comment (sypets) [reply README.md:16]: Sure. That was just a suggestion. (As extension author you already have to update the version constraints in several places, e.g. ext_emconf.php, composer.json etc. I personally find it a nuisance and in practice often this gets neglected.)

Actually it's a little beyond the scope of your PR anyway - so just ignore my comment for now. Thanks and thanks for fixing the README.

**Review by sypets — APPROVED** [review]

**Comment by NairCoder** [comment]
Merged to master with minor corrections. Hence closing this MR. Thanks @ayacoo @sypets

**Pull request closed without merging**